src/main/scala/nl.knaw.dans.easy.deposit/DepositDir.scala

```diff
@@ -229,6 +229,7 @@ object DepositDir {
       addProperty("curation.performed", "no")
       addProperty("identifier.dans-doi.registered", "no")
       addProperty("identifier.dans-doi.action", "create")
+      addProperty("bag-store.bag-name", "bag")
```

Comment:
is there some kind of constant or function with this name. Given that it is a hardcoded name here, indicates it should be somewhere where the bag is created as well.

@jo-pol sidenote: I would rather have a better name for the bag directory. Do you have any suggestions?

Reply:
Introduce a constant also to be used at https://github.com/DANS-KNAW/easy-deposit-api/blob/e0ddf23b3ac32591233306e15de0e112e5e4b0c5/src/main/scala/nl.knaw.dans.easy.deposit/DepositDir.scala#L45

The property name is indeed confusing for me in this context but is related to https://github.com/DANS-KNAW/easy-manage-deposit/pull/51

Reply:
To make it more complicated: 

Though the values are the same, you should share a constant with https://github.com/DANS-KNAW/easy-deposit-api/blob/c51c58a3b6bd24d87e295bf6dfef1f4ab84c2466/src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala#L92 in theory the value on line 45 could be different...